robottelo/common/helpers.py

```diff
@@ -302,3 +302,34 @@ def patched_computeresource_init(self, server_config=None, **kwargs):
             computeresource_init(self, server_config, **kwargs)
             self._fields['url'].default = docker_url
         entities.ComputeResource.__init__ = patched_computeresource_init
+
+
+def prepare_import_data(tar_path=None):
+    """Fetch and uncompress the CSV files from the source."""
+    tmpdir = '/tmp/'
+    # if path to tar file not specified, download the default one
+    if tar_path is None:
+        tar_remote_path = conf.properties['transitions.export_tar.url']
+        cmd = u'wget {0} -O - | tar -xzvC {1}'.format(tar_remote_path, tmpdir)
+    else:
+        cmd = u'tar -xzvf {0} -C {1}'.format(tar_path, tmpdir)
+
+    files = {}
+    for file in ssh.command(cmd).stdout:
+        if file.endswith('activation-keys.csv'):
+            files['activation-keys'] = tmpdir + file
+        elif file.endswith('channels.csv'):
+            files['channels'] = tmpdir + file
+        elif file.endswith('config-files-latest.csv'):
+            files['config-files-latest'] = tmpdir + file
+        elif file.endswith('kickstart-scripts.csv'):
+            files['kickstart-scripts'] = tmpdir + file
+        elif file.endswith('repositories.csv'):
+            files['repositories'] = tmpdir + file
+        elif file.endswith('system-groups.csv'):
+            files['system-groups'] = tmpdir + file
+        elif file.endswith('system-profiles.csv'):
+            files['system-profiles'] = tmpdir + file
+        elif file.endswith('users.csv'):
```

Comment:
yeah, sorry. There are alternatives anyway. Just such long if statements are not readable at all. I prefer @Ichimonji10 approach more